runtimes/ruby-3.0/README.md

```diff
@@ -18,7 +18,7 @@ printf 'def main(request, response)\n    return response.json({:n => rand()})\ne
 2. Build the code:
 
 ```bash
-docker run --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:3.0 sh /usr/local/src/build.sh
+docker run -e ENTRYPOINT_NAME=index.rb --rm --interactive --tty --volume $PWD:/usr/code openruntimes/ruby:3.0 sh /usr/local/src/build.sh
```

Comment:
Why do we need a `ENTRYPOINT_NAME` 

1. build.sh does not seem to be using this env variable anywhere
2. Didn't we rename it to `INTERNAL_RUNTIME_ENTRYPOINT`  ??

Please check this in the other files as well.